data/Makefile.am

```diff
@@ -2,7 +2,7 @@ CLEANFILES =
 NULL =
 
 desktopdir=$(datadir)/applications
-desktop_DATA = org.gnome.Shell.desktop gnome-shell-extension-prefs.desktop eos-launch.desktop
+desktop_DATA = org.gnome.Shell.desktop gnome-shell-extension-prefs.desktop eos-launch.desktop eos-link-feedback.desktop
```

Comment:
This commit is from @GeorgesStavracas, not sure how it ended up in this PR (should be gone now)

In any case, I agree with you that this should be moved to Yelp